CHANGELOG.asciidoc

```diff
@@ -29,6 +29,17 @@ See Conventional Commits (https://conventionalcommits.org) for commit guidelines
 [[release-notes-5.x]]
 === RUM JS Agent version 5.x
 
+[[release-notes-5.11.1]]
+==== 5.11.1 (2022-05-16)
+[float]
+===== Features
+* Span's outcome is now set to unknown when a fetch/xhr request aborts. The outcome can be found in the metadata information of the span details in APM UI: {pull}1211[#1211]
+* Allow to [disable events payload compression](https://www.elastic.co/guide/en/apm/agent/rum-js/current/troubleshooting.html#disable-events-payload-compression). This lets to inspect them for debugging purposes as well as for creating HAR files making sure they are readable: {pull}1214[#1214]
```

Comment:
I would reword it in this way. 

+ Allow users the ability to disable compression for the events sent to APM server. This is helpful for debugging purposes while inspecting via Devtools and also for generating HAR file.